app/src/main/java/com/pec_acm/moviedroid/data/api/TMDBApi.kt

```diff
@@ -90,4 +87,12 @@ interface TMDBApi {
         @Query("api_key")
         apiKey : String = Constants.API_KEY,
     ) : Response<TVDetail>
+
+    @GET("movie/{movie_id}/videos")
+    suspend fun movieTvVideosByID(
+        @Path("movie_id") movie_id : Int,
+        @Query("api_key")
+        apiKey : String = Constants.API_KEY
+    ) : Response<MovieTvVideo>
+
```

Comment:
This endpoint is only for movies, there is a separate endpoint for [tv shows](https://developers.themoviedb.org/3/tv/get-tv-videos). Just create a separate function for that and make changes in other files accordingly, also change the names of the existing functions accordingly indicating that they are for movies and not tv shows 😊

Reply:
Yeah good catch , 

actually i noticed that there was a different endpoint for tv shows and the thing is i reused the same apiRespose data class , but forgot to create a different get method for the tv endpoint 😅 . Will fix that in a bit .

Thanks for confirming 👍